src/views/PageWrapper/PageWrapper.tsx

```diff
@@ -26,7 +28,7 @@ const PageWrapper: React.FC<Props> = ({ children }: Props) => {
       <TopToolbar />
       <PageBody>
         <RightPanel />
-        {children}
+        <UIOptionsPopup>{children}</UIOptionsPopup>
```

Comment:
If you use a button triggered modal, as suggested later, no need to pass the page body children into this component for the context menu trigger.

Reply:
I have submitted changes to honor your suggestions!